main.go

```diff
@@ -89,13 +89,14 @@ func main() {
 	}
 	ds, err := NewPgReplicationDataSource(cfg)
```

Comment:
Is there a way we can make a new pg replication data source not return an error when initialized?